ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -516,8 +845,8 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
     ComponentTransportProcessData const& _process_data;
 
     IntegrationMethod const _integration_method;
-    std::vector<std::reference_wrapper<ProcessVariable>> const&
-        _per_process_variables;
+    std::vector<std::reference_wrapper<ProcessVariable>> const
+        _transport_process_variables;
```

Comment:
So, these are only the concentration variables, aren't they?
And therefore the hydraulic process variable is stored somewhere else.

Reply:
In my opinion, picking hydraulic process variable out from the process variables as well is not necessary.